lib/ejs.js

```diff
@@ -170,7 +171,11 @@ function includeFile(path, options) {
   if (!opts.filename) {
     throw new Error('`include` requires the \'filename\' option.');
   }
-  opts.filename = exports.resolveInclude(path, opts.filename);
+  if(path.substr(0,1) === "/" && typeof opts.root == "string" && opts.root != ""){
```

Comment:
Is there a real risk people will pass a non-string as root? Also, why the strict equality check against `"/"`? Could probably just do this:

```javascript
if (/^\//.test(path) && opts.root) { ...
```